features/taxation/applying_taxes/applying_correct_taxes_for_order_with_promotion_giving_discount_for_and_item_in_order_when_tax_rate_included_in_price.feature

```diff
@@ -0,0 +1,50 @@
+@applying_taxes
+Feature: Apply correct taxes on order item level when order has applied a promotion with discount for an item in order and tax rate is included in price
```

Comment:
And it is a little bit to verbose for me
`Apply correct taxes for order with item discount and tax rates included in price`